src/lib.rs

```diff
@@ -248,11 +250,19 @@ where
         let thread = std::thread::current();
         let thread_name = thread.name();
 
+        #[allow(unused_assignments, unused_mut)]
+        let mut metadata = None;
+
+        #[cfg(feature = "tracing-log")]
+        {
+            metadata = event.normalized_metadata();
+        }
```

Comment:
I'll fix this after, but I don't think we should allow unused mut/assignment. i'll make this fix.

Reply:
done.